File: server/core/enums.ts
export enum LoginStatus {
  error,
  invalid,
  denied,
  success,
}

export enum InvoiceNumberStatus {
  success,
  fail,
  noRef,
}

export enum RestoreProdStatus {
  restored,
  failed,
  noRef,
}

export enum ViewPasswordStatus {
  success,
  noRef,
  noPasswordSet,
  failed,
}

export enum SetPasswordStatus {
  failed,
  noRef,
  success,
}

File: client/lib/core/enums.dart
enum LoginStatus { error, invalid, denied, success }

enum InvoiceNumberStatus { success, fail, noRef, unauthorized }

enum RestoreProdStatus { restored, failed, noRef, unauthorized }

enum ViewPasswordStatus { success, noRef, noPasswordSet, failed, unauthorized }

enum SetPasswordStatus { failed, noRef, success, unauthorized }

enum UpdateSubscriptionStatus { success, failed, noRef, unauthorized }

enum CreateAccountStatus { success, failed, alreadyRegistered, unauthorized }
